client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -89,8 +89,9 @@ const StatsInsights = ( props ) => {
 							hideNewModule // remove when cleaning 'stats/horizontal-bars-everywhere' FF
 						/>
 
+						{ /* Replaced by new modules on top of the page */ }
 						{ ! isLatestPostReplaced && <LatestPostSummary /> }
-						<Reach />
```

Comment:
I know it's part of our design specifications, but I don't feel comfortable getting rid of the `Subscriber totals` card without preserving its functionality somewhere else in the interface.

I know WPCOM and Email subscriber totals remain accessible via the `Subscribers` card, but social subscriber total isn't easily obtainable in the interface without manual summation.

What are your thoughts?

Reply:
The email subscriber total is not included in the year in review. 
Another card at the top could be helpful, aggregating totals for many items.
We could also extend the component to show some footer with additional information and put the totals back on the cards.
The second option is better because it doesn't push down the content that was previously at the top of the page.

Reply:
I looked into the components further, and since we preserve the information about total wp.com subscribers as part of `Social subscribers(followers)`, we could maintain the information about social networks by adding a row that can be expanded into a list of networks (just like we do on the Traffic page with Authors).

Reply:
I also like how much cleaner the interface is. Your observation is correct; the subscribers count in the annual highlights section counts WPCOM subscribers and is already counted here as well:

<img width="414" alt="image" src="https://user-images.githubusercontent.com/4044428/217947917-a3be069a-1f19-415c-ad42-504a02fc0089.png">

Reply:
At any rate, we can iterate on this refinement next week during our maintenance sprint 🙂